shared/AppInsightsCore/src/JavaScriptSDK/BaseCore.ts

```diff
@@ -181,4 +196,19 @@ export class BaseCore implements IAppInsightsCore {
 
         return new ProcessTelemetryContext(thePlugins, _this.config, _this);
     }
+
+    areExtensionsInitialized(): boolean {
+        let _this = this;
+        if (!_this._allExtensionsInitialized) {
+            let extensions = _this._extensions;
```

Comment:
During BaseCore initialization, this actually, won't include all of the extensions after initialization, it will only contain those "managed" by BaseCore and not the ones that have been delegated to the ChannelController

Reply:
Removed this method entirely